### PR TITLE
config: add Sentry

### PR DESCRIPTION
**Pull request by hdonnay**
Depends on #1870

### CODE REVIEW

**Comment by hdonnay** [comment]
Pretty sure these are the only parts of the [Sentry SDK] we want to expose directly. There's already a sampling knob, and we should be able to cook up a way to get sample-only-if-marked working.

[Sentry SDK]: https://pkg.go.dev/github.com/getsentry/sentry-go#ClientOptions

File: config/introspection.go [hunk 1]
```diff
@@ -50,6 +52,17 @@ type Jaeger struct {
...
 
+// Sentry is the [Sentry] specific tracing configuration.
+//
+// [Sentry]: https://sentry.io
+type Sentry struct {
+	// DSN to be passed to [github.com/getsentry/sentry-go.ClientOptions.Dsn].
+	DSN string `yaml:"dsn" json:"dsn"`
+	// Environment to be passed to
+	// [github.com/getsentry/sentry-go.ClientOptions.Environment].
+	Environment string `yaml:"environment,omitempty" json:"environment,omitempty"`
```
Comment (crozzy) [inline config/introspection.go:63]: Presumably this will be `stage`, `prod` etc?

Comment (crozzy) [reply config/introspection.go:63]: Regarding `Debug`, do we want to allow users to toggle that or could we eventually key it off of `clair.config.log_level`.

Comment (crozzy) [reply config/introspection.go:63]: I don't think anything else makes sense, `dist` I'd imagine is something we want to describe the Clair version (and all it's facets).

Comment (hdonnay) [reply config/introspection.go:63]: I think the `Debug` spews stuff out to a configured writer, so we'd have to cook something up to repackage that into our logs. I think I can/will add this if it ends up being needed.

Comment (hdonnay) [reply config/introspection.go:63]: That is to say: yes, we should control that via the main logging tunable, but it'll complicate the integration a little bit.

**Review by crozzy — APPROVED** [review]
LGTM

**Pull request merged**